README.md

```diff
@@ -74,10 +74,17 @@ or a [Jupyter notebook](https://docs.jupyter.org/en/latest/running.html), and tr
 ``` python
 import pygmt
 fig = pygmt.Figure()
-fig.coast(projection="H10c", region="g", frame=True, land="gray")
+fig.coast(projection="N10c", region="g", frame=True, land="tan", water="lightblue")
+# Add a semi-transparent text element
+fig.text(position="MC", text="PyGMT", font="60p,Helvetica-Bold,red@75")
 fig.show()
 ```
 
+You should see a global map with land and water masses colored in tan and lightblue, respectively. On top, there
+should be the semi-transparent text "PyGMT". If the semi-transparency does not show up, there is  probably an
+incompatibility between your GMT and ghostscript versions. For details, please run `pygmt.show_versions()` and
+see [Not working transparency](https://www.pygmt.org/dev/install.html#not-working-transparency).
+
```

Comment:
I prefer to keep the README file (and also the PyGMT landing page) as simple as possible:
```suggestion
You should see a global map with land and water masses colored in tan and lightblue, respectively, with a semi-transparent text "PyGMT" on top. 
```